src/components/ViewerNavigation.js

```diff
@@ -41,6 +41,7 @@ export class ViewerNavigation extends Component {
 
     return (
       <div
+        role="navigation"
```

Comment:
I'll have to go back and look, but I dimly recall we intentionally decided that a navigation role was inappropriate here.

Reply:
Ok. I'm not sure how else to select that item. The first test might not be necessary (just checking for existence of the wrapper `<div>`). However, there's another test that is specifically checking the viewing direction....
https://github.com/ProjectMirador/mirador/pull/3685/files#diff-69ebf93942b44352912effb10c94e7ae4687632918a4a61526ea1e37a9e7dbdcR122

Reply:
@cbeer I updated the tests to not rely on the navigation role. I removed the navigation role from the component.

https://github.com/ProjectMirador/mirador/pull/3685/commits/967dea90aeb2bfba0d684bb8b3c12e878dde7ebd